frontend/App.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { render } from 'react-dom';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import LinkItem from './components/LinkItem';
+import TokenPage from './FullPageRoutes/TokenPage';
+import DashboardPage from './FullPageRoutes/DashboardPage'
+
+
+function MainPage() {
+    return (
+        <ul>
+            <LinkItem to='/Tokens' text='Tokens' />
+            <LinkItem to='/Dashboard' text='Dashboard' />
+        </ul>
+    );
+}
+
+export default function App() {
+    return (
+        <Router>
+            <Switch>
+                <Route exact path='/Dashboard'>
+                    
+                    <DashboardPage />
+                </Route>
+                <Route path='/Tokens'>
+                    <TokenPage />
+                </Route>
+                <Route path='/'>
+                    <MainPage />
+                </Route>

```

Comment:
I think it's safe to say that all of these should be exact. I'm not 100% certain about the `/` path. But let's try it